.github/workflows/main.yaml

```diff
@@ -257,9 +267,17 @@ jobs:
         uses: ./.github/actions/gn
       - name: Build Cobalt
         uses: ./.github/actions/build
+      - name: 'Check Artifact Size'
+        uses: ./.github/actions/check_artifact_size
+        if: ${{ inputs.check_artifact_size && matrix.sb_api_version == ''}} # Only check current SB version for Evergreen.
```

Comment:
Instead of adding `inputs.check_artifact_size` you could just check that the `evergreen_loader` platform variable is set: `if: ${{ needs.initialize.outputs.evergreen_loader != 'null' && matrix.sb_api_version == ''}}`.

Then the comment can be removed too since the line is self explanatory.

Reply:
Ideally, we should perform a check on `inputs.check_artifact_size` only to gate this action in a platform-agnostic manner. In the future, we can utilize this flag for other platforms.

I added the Evergreen check here because I found that `libcobalt.so` is not generated in out/${{ matrix.platform }}_${{ matrix.config }} when building older SB versions on GitHub CI due to some reason. This can be demonstrated in this  snippet: [No files were found with the provided path: out/evergreen-x64-sbversion-14_devel/libcobalt.so](https://github.com/youtube/cobalt/actions/runs/9981768009/job/27586213962?pr=3850#step:8:33)

Reply:
Update: I was searching the wrong path for old SB versions, will be able to modify it to support all SB versions.